lib/percy/client/connection.rb

```diff
@@ -91,6 +91,10 @@ def post(path, data, options = {})
             request.body = data.to_json
           end
         rescue Faraday::TimeoutError
+          if (retries -= 1) >= 0
+            sleep(rand(1..3))
```

Comment:
I'd actually be comfortable to drop this sleep entirely, considering our current understanding of the load balancing path.